src/coffee/run.coffee

```diff
@@ -213,6 +212,7 @@ module.exports = class
         .then (authConfig) =>
           options =
             timeout: program.timeout
+            exportFormat: 'log'
```

Comment:
I cant see where this variable is actually used?

Reply:
Not used, will remove it.